### DISCUSSION TITLE
how do I create a did:web based account on bluesky's PDS?

### DISCUSSION THREAD

**Discussion by TheButlah** [bluesky-social/atproto#3140] [How to do it?]
I have seen guides on how to migrate *from* bluesky's PDS to a self hosted pds. Also guides on how to migrate from a did:plc to a did:web on a self hosted PDS.

But how do I create a did:web (for example, on bluesky's own PDS?

**Comment by somini** [comment]
Yes, I would like a tutorial for this. From what I can tell, with `did:web` I can fully own the identity, but use `bsky.social` PDS instead of running my own.

I only found this: https://github.com/savely-krasovsky/bsky-did-web

I followed that using `bksy.social` as a PDS, but I cannot get an invite code.

**Comment by yamarten** [comment]
bsky.social does not guarantee that did:web account creation will work.

The flow of creating an account without creating a new DID is the same as [PDS migration](https://github.com/bluesky-social/pds/blob/main/ACCOUNT_MIGRATION.md), except for import data. Due to technical problems, they have not yet supported it.

https://whtwnd.com/did:plc:44ybard66vv44zksje25o7dz/3l5ii332pf32u

For self-hosting PDS, #2100 is a reference. To use [bsky-did-web](https://github.com/savely-krasovsky/bsky-did-web) on a PDS without an invitation code, you probably just omit that option.
